user-profiles.md

```diff
@@ -0,0 +1,33 @@
+# User Profiles
+
+When considering requirements and implementation solutions it's useful look at who uses Helm to have a better understanding of how something should work. The Helm user profiles describe the different types of users and contributors to Helm along with the order of priority they have relative to each other. The ordering is because nothing can share the exact same priority and we want to convey that.
+
+## Profiles
+
+Profiles describe a type of user. A real person may perform more than one type of role.
+
+### 1. Application Operator
```

Comment:
I think it's easy to get this term confused with Cluster Operator. In my view, this use-case is when you don't really care how the app is provisioned inside the cluster, but you want to be able to easily install and manage these apps at a higher-level. If my assumption is correct there, should we maybe distinguish that this would not be the same as a Cluster Operator?

Of course, if this has already been discussed at length at the Summit, then feel free to ignore my comment :P.

Reply:
This is a good catch. I added a note and comment to this section. Let me know if that works.

Reply:
lgtm, thanks!